server/games/game.py

```diff
@@ -87,6 +87,10 @@ class ValidityState(IntEnum):
     UNLOCKED_TEAMS = 17
     MULTI_TEAM = 18
     HAS_AI_PLAYERS = 19
+    CIVILIANS_REVEALED = 20
+    TOO_EASY = 21
```

Comment:
WRONG_DIFFICULTY please 😛

Reply:
Ok. Was trying to be consistent, but wrong difficulty is probably better.